### ISSUE TITLE
Put a (temporary) ban on authors of poor quality katas

### ISSUE THREAD

**Issue by docgunthrop** [Codewars/codewars.com#1118]
I've come across a couple rare instances where a kata author will publish a broken or very low quality kata and leave it as is without ever fixing any outstanding issues (ie. a "set it and forget it" approach).

I suggest implementing a temporary ban on such authors, preventing them from publishing any further katas in order to mitigate further pollution of the current kata pool. This could probably be done by taking into account the number of dissatisfied votes and issue reports for a given kata.

**Comment by jhoffner** [comment]
Do we have evidence that these poor authors continue to create multiple kata **after** the point in which we could determine that their previous kata are poor quality?

**Comment by jhoffner** [comment]
Perhaps it would be better to simply not allow any author to publish a new kata unless all of their open issues are closed.

**Comment by Voileexperiments** [comment]
Maybe instead we should:

- Raise the limit for writing new katas. `75` honor is definitely too low.
- Give the ability to *retire* beta katas, to stop cluttering stuff
- Smack people with the "writing your first kata" docs page in their face when they enter the kata editor without any published kata. I spent lots of efforts writing that docs and if I must say so, that should cover pretty much all of the hot garbage sources.

**Comment by docgunthrop** [comment]
Definitely agree about raising the minimum honor limit for publishing a kata.

With regard to @Voileexperiments third bullet point : it could be similar to what you see when posting a question on Stack Overflow; the page provides a checklist of prerequisites to cover before posting. This list includes things like checking for duplicates and other rules of thumb to follow. Perhaps a summarized version of the "writing your first kata" docs page with a link to the full page included, if the full page is too long.

@jhoffner I completed a kata today that appears to have had no effort put into making sure it worked. The tests didn't work, the variable names were wrong, and the description was incomplete. I checked the author's previously published katas and there was one from a week before that was a very very low effort kata which still has outstanding issues/suggestions from when it was published.

**Comment by anter69** [comment]
I totally agree with raising the bar for kata authoring: I strongly support all points of @Voileexperiments.

I also agree for the temporary ban, but with some finetuning:
* consider restricting it to beta katas only -- once a kata is live, the issues are usually not really blocking ones
* take into account translations somehow -- otherwise you may block a "good" author, just because he accepted a low quality translation
* compare the number of issues to the total number of katas authored by the user -- 5 open issues in 100 katas is definitely not as bad as 5 issues in 1 kata... (think of @myjinxin2015, e.g.)

**Comment by Voileexperiments** [comment]
@docgunthrop Actually, maybe we should first fix the bug where the initial publish does not trigger validation.

Actually, it seems that *somehow* I haven't raised an issue about that. I'll do it now.
